### DISCUSSION TITLE
Suggestion for Programmatic Layer Control in Animation Model

### DISCUSSION THREAD

**Discussion by rradnev1** [NRC-Research/SNAP-issues#84] [Ideas]
Dear SNAP development team,

I'd like to suggest adding programmatic layer control to the animation model. This would allow users to change the layer of drawn elements using Python scripting or a similar mechanism.

This enhancement would enable more dynamic and interactive visualizations, such as animations where components automatically appear or disappear based on certain conditions.

Sincerely,
Radko

**Comment by NRCgg** [comment]
@WDunsford This is a discussion that you probably should be aware of.
